builtin/providers/aws/cloudfront_distribution_configuration_structure_test.go

```diff
@@ -44,6 +49,35 @@ func trustedSignersConf() []interface{} {
 	return []interface{}{"1234567890EX", "1234567891EX"}
 }
 
+func lambdaSetHash(v interface{}) int {
```

Comment:
This is essentially where my PR was at [9c5a25d](https://github.com/hashicorp/terraform/pull/11291/files/9c5a25dd480e71a3b0838c4c2f3f1cae47f41757) on Jan 5. I was derailed by the [comment from @apparentlymart](https://github.com/hashicorp/terraform/pull/10985#discussion_r94830032) asking to remove the hash function.

Sorry this got delayed 2 weeks as a result 🤓